sdcm/nemesis.py

```diff
@@ -934,13 +997,16 @@ class NodeToolCleanupMonkey(Nemesis):
     def disrupt(self):
         self.disrupt_nodetool_cleanup()
 
+
+@run_in_mixed_mode
```

Comment:
Why do we need the decorator on the class?

Reply:
I think ChaosMonkey should be marked, because it could call disrupt methods which will not related to spot instances.
Or we can change the decorator from Nemesis classes to methods of Nemesis class, then only specific methods will be affected